README.md

```diff
@@ -14,12 +14,32 @@ $ docker run --link some_mongo_container:mongo -p 8081:8081 -e ME_CONFIG_MONGODB
 
 Then you can hit `http://localhost:8081` or `http://host-ip:8081` in your browser.
 
+Some alternatives for the `ME_CONFIG_MONGODB_URL` environment variable:
+- `mongodb://host.docker.internal:27017`
+- `mongodb://host.containers.internal:27017`
+
 ## Security Notice
 
 JSON documents are parsed through a javascript virtual machine, so the web interface can be used to execute malicious javascript on a server.
 
 **mongo-express should only be used privately for development purposes.**
 
+# Build
+Build the mongo-express image:
+```console
+docker build --tag mongo-express .
```

Comment:
Instead of using "." should we use an example path to one of the docker context folders? We should assume the user is running from the root folder. It for this example maybe it makes sense to just use the most recent alpine and node one?

Reply:
```suggestion
docker build --tag mongo-express 1.0/20-alpine3.18
```